js/apps/system/_admin/aardvark/APP/frontend/scss/_modals.scss

```diff
@@ -519,6 +519,24 @@
     margin-bottom: 0;
     width: 100%;
   }
+
+  .moveRowButton {
+    margin-left: 5px;
+    font-size: 14pt;
+
+    &:hover {
+      cursor: pointer;
+      color: rgba(0,0,0,.8);
+    }
+
+    &.disabled {
+      color: rgba(0,0,0,.3);
+
+      &:hover {
+        cursor: default;
+      }
+    }
+  }
```

Comment:
```suggestion
```

Reply:
Same as above - we don't need move functionality anymore?

Reply:
Due to technical issues with the old framework, this feature should be added in our upcoming refactoring using the react js framework. This feature is planned for 3.10. The required time for implementing this feature in 3.9 (using backbone) only, and then drop it in 3.10 anyway, is not worth it. Additionally, a 3.9 backbone based implementation would have been super ugly, from a technical point of view. Also, the required time is out of proportion compared to the benefits it would deliver here.